Detach.txt

```diff
@@ -0,0 +1,45 @@
+ - Detach Market Apps Configuration -
+
+ - Remove comment (#) to detach an App.
+ 
+#Contacts 
+#Gmail 
+#Google App 
+#Google Plus 
+#Hangouts 
+#Phone 
+#Photos 
+YouTube
+#YouTube Music
```

Comment:
It's useless to adding another app here if you don't edit the `customize.sh` file..

Reply:
I did not updated here

Reply:
> in test repo. I'll push the change to main repo after completing the testing

Np. Take your time.